index.html

```diff
@@ -0,0 +1,69 @@
+<!-- <img class="Ellipse12" src="images/Ellipse 12.png" alt="">
+            <img class="Ellipse13" src="images/Ellipse 13.png" alt="">
+            <img class="Ellipse4" src="images/Ellipse 4.png" alt="">
+            <img class="Rectangle5" src="images/Rectangle 5.png" alt="">
+            <img class="Ellipse3" src="images/Ellipse 3.png" alt="">
+            <img class="Rectangle4" src="images/Rectangle 4.png" alt="">
+            <img class="Rectangle7" src="images/Rectangle 7.png" alt="">
+            <img class="Ellipse5" src="images/Ellipse 5.png" alt="">
+            <img class="Ellipse11" src="images/Ellipse 11.png" alt="">
+            <img class="Rectangle6" src="images/Rectangle 6.png" alt="">
+            <img class="Ellipse15pro" src="images/Ellipse15pro.png" alt="">-->
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <style>
+    
+    </style>
+</head>
+<body>
+
+        <header class="header">
+            <a class="logo" href="#"><img src="images/My Logo.png" alt=""></a>
+            <nav class="navbar">
+                <a href=""><img src="images/Union.png" alt=""></a>
+            </nav>
+        </header>
+
+        <div id="bg-Img">
+
+        <section class="main">
+            <div class="heading">
+                <h1>I'm Alex. <br>
+                    Glad to see you!</h1>
+
+
+                    <p>
+                        I'm a software developer! I can help you build a product , 
+                        feature or website Look through some of my work and experience!
+                         If you like what you see and have a project you need coded, 
+                         don't hesitate to contact me.
+                    </p>
+                
+                    <a href="#">LET'S CONNECT</a>
+            </div>
+
+            
+            <ul>
+                <li><a href="#" class="twitter" ><img src="images/twitter.svg" alt=""></a></li>
+                <li><a href="#" class="linkedin" ><img src="images/Linkedin-icon.svg" alt=""></a></li>
+                <li> <a href="#" class="vectorM" ><img src="images/VectorM.png" alt=""></a></li>
+                <li><a href="#" class="github" ><img src="images/github.svg" alt=""></a></li>
+                <li><a href="#" class="vectorsixer" ><img src="images/Vectorsixer.png" alt=""></a></li>
+               </ul>
+
+            </div>
+
+        </section>
+
+</body>
+</html>
```

Comment:
[OPTIONAL] Your html code is difficult to read and understand since it is not well formatted. Kindly consider using a code formater like prettier to make it well arranged and easy to read and understand